### PR TITLE
feat: #4688 geo diary marketing content

### PR DESCRIPTION
**Pull request by willmcvay**
# Pull request checklist

**Detail as per issue below (required):**

fixes: #4688

### CODE REVIEW

File: packages/geo-diary-marketing/src/components/compatibility-slide.svelte [hunk 1]
```diff
@@ -22,34 +22,34 @@
...
-    max-width: 1200px;
+  .container {
+    margin-bottom: 2rem;
   }
 
   .padded-container {
     padding-top: 2rem;
     background: rgb(248, 248, 248);
-    width: 100%;
+    max-width: 1440px;
```
Comment (bashleigh) [inline packages/geo-diary-marketing/src/components/compatibility-slide.svelte:52]: variable?

File: packages/geo-diary-marketing/src/components/compatibility-slide.svelte [hunk 2]
```diff
@@ -103,13 +109,13 @@
     font-size: 0.875rem;
   }
 
-  @media screen and (max-width: 1100px) {
+  @media screen and (max-width: 2880px) {
```
Comment (bashleigh) [inline packages/geo-diary-marketing/src/components/compatibility-slide.svelte:112]: variable?  non-response breakpoint?

File: packages/geo-diary-marketing/src/components/compatibility-slide.svelte [hunk 3]
```diff
@@ -103,13 +109,13 @@
...
     .image-desktop-hero-container {
       display: none;
     }
   }
 
-  @media screen and (min-width: 1099px) {
+  @media screen and (min-width: 2879px) {
```
Comment (bashleigh) [inline packages/geo-diary-marketing/src/components/compatibility-slide.svelte:118]: variable? non-response breakpoint?

File: packages/geo-diary-marketing/src/components/features-slide.svelte [hunk 4]
```diff
@@ -10,8 +18,8 @@
         <div class="tile-content">
           <h6 class="el-subtitle el-w1_2_2 el-text-center">Send ETA</h6>
           <p class="el-body-text el-has-grey-text is-hidden-mobile">
-            When you are ready to publish your app you will need to add some additional information such as a Summary,
-            Description & Images and you will also need to make it 'Listed'.
+            Send a prepopulated text message with your exact time of arrival. GEO Diary will automatically calculate
+            your current location and the time to your next appointment.
```
Comment (bashleigh) [inline packages/geo-diary-marketing/src/components/features-slide.svelte:22]: the time? As in time of arrival?

File: packages/geo-diary-marketing/src/components/features-slide.svelte [hunk 5]
```diff
@@ -22,8 +30,8 @@
         <div class="tile-content">
           <h6 class="el-subtitle el-w1_2_2 el-text-center">Location Settings</h6>
           <p class="el-body-text el-has-grey-text is-hidden-mobile">
-            When you are ready to publish your app you will need to add some additional information such as a Summary,
-            Description & Images and you will also need to make it 'Listed'.
+            Get turn by turn directions from any starting location. The app will automatically detect your current
+            location but it also provides the ability to set a new location.
```
Comment (bashleigh) [inline packages/geo-diary-marketing/src/components/features-slide.svelte:34]:
```suggestion
            location but it also provides the ability to set a different location.
```
Just a suggestion?

File: packages/geo-diary-marketing/src/components/intro-slide.svelte [hunk 6]
```diff
@@ -8,22 +8,24 @@
       <div class="column el-flex-container el-flex-column el-flex-justify-evenly el-flex-grow no-margin">
         <img class="logo-image" src="./images/intro-title.svg" alt="Geo Diary logo" />
         <div class="el-mb6">
-          <h6 class="el-subtitle">Lets get started!</h6>
+          <h6 class="el-subtitle">Access your appointments, anywhere!</h6>
           <p class="el-body-text el-has-grey-text">
-            You can also use the listing preview to see how your app will display in the Marketplace to potential
-            clients.
+            Simple and easy to use. The GEO Diary app allows you to easily access your diary appointments dire¬ct from
```
Comment (bashleigh) [inline packages/geo-diary-marketing/src/components/intro-slide.svelte:13]:
```suggestion
            Simple and easy to use. The GEO Diary app allows you to easily access your diary appointments direct from
```

Is this simple a tab or new line? Looks like a tab to me!

File: packages/geo-diary-marketing/src/components/intro-slide.svelte [hunk 7]
```diff
@@ -8,22 +8,24 @@
...
+            your mobile or web browser. Why not see it for yourself?
```
Comment (bashleigh) [inline packages/geo-diary-marketing/src/components/intro-slide.svelte:14]:
```suggestion
            your mobile or web browser. Why not see for yourself?
```

File: packages/geo-diary-marketing/static/styles/pardot.css [hunk 8]
```diff
@@ -31,16 +38,25 @@ email capture - uploaded manually via salesforce - just storing them here for fu
...
   outline-color: #262f69;
+}
+
+#pardot-form .errors {
+  display: none;
+}
+
+#pardot-form .error {
+  display: none;
 }
```
Comment (bashleigh) [inline packages/geo-diary-marketing/static/styles/pardot.css:62]: EOF

**Review by bashleigh — CHANGES_REQUESTED** [review]
Looks good! Just added some basic suggestions and a few question/suggestions :)

File: packages/geo-diary-marketing/src/components/compatibility-slide.svelte (hunk 1 above)
Comment (willmcvay) [reply packages/geo-diary-marketing/src/components/compatibility-slide.svelte:52]: done

File: packages/geo-diary-marketing/src/components/compatibility-slide.svelte (hunk 2 above)
Comment (willmcvay) [reply packages/geo-diary-marketing/src/components/compatibility-slide.svelte:112]: As discussed, you can't use css vars in media queries am afraid

File: packages/geo-diary-marketing/src/components/compatibility-slide.svelte (hunk 3 above)
Comment (willmcvay) [reply packages/geo-diary-marketing/src/components/compatibility-slide.svelte:118]: As above

File: packages/geo-diary-marketing/src/components/intro-slide.svelte (hunk 6 above)
Comment (bashleigh) [reply packages/geo-diary-marketing/src/components/intro-slide.svelte]: symbol, jesus I was asleep

**Review by bashleigh — APPROVED** [review]

**Pull request merged**